experiments/HSA_20150714_MI/hsa_MI_20150714.py

```diff
@@ -0,0 +1,291 @@
+#! /usr/bin/env python
+"""
+Script for generation of input files for Aspirin and Naproxen binding to HSA by ITC.
+"""
+
+from itctools.itctools import ureg, Quantity
+from itctools.procedures import ITCProtocol, ITCExperimentSet, ITCExperiment, ITCHeuristicExperiment
+from itctools.materials import Solvent, Compound, SimpleSolution
+from itctools.labware import Labware, PipettingLocation
+import pprint
+
+
+## Is this volume correct? I can try adding units.
+
+# The sample cell volume in microliters
+cell_volume = 202.8
+
+# Define solvents.
+water = Solvent('water', density=0.9970479 * ureg.gram / ureg.milliliter)
+buffer = Solvent('buffer', density=1.014 * ureg.gram / ureg.milliliter) # TODO is our density the same as the HOST-GUEST buffer?
+
+# Define compounds.
+hsa = Compound('HumanSerumAlbumin', molecular_weight=65000 * (ureg.gram / ureg.mole), purity=.95)
+naproxen_sodium = Compound('Naproxen Sodium', molecular_weight=252.24 * (ureg.gram / ureg.mole), purity=1.)
+
+
+## Aspirin can be insoluble? Are we using its salt?
```

Comment:
Aspirin has an aqueous solubility of 1 g / 300 mL water, which I think comes to ~18 mM.